specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2020-03-15-preview/storagepool.json

```diff
@@ -1023,19 +1371,20 @@
         "username",
         "password"
       ],
-      "description": "CHAP credentials for an iSCSI target",
+      "description": "Challenge Handshake Authentication Protocol (CHAP) credentials for an iSCSI target ACL.",
       "properties": {
         "username": {
           "type": "string",
-          "description": "Username for Challenge Handshake Authentication Protocol (CHAP) authentication",
+          "description": "Username for Challenge Handshake Authentication Protocol (CHAP) authentication.",
           "minLength": 7,
-          "maxLength": 128
+          "maxLength": 511
         },
         "password": {
```

Comment:
Mark this property as a secret, see here:

https://armwiki.azurewebsites.net/rp_onboarding/process/property_design_best_practices.html?q=x-ms-secret

Reply:
We mark the entire object as a secret on this [line](https://github.com/Azure/azure-rest-api-specs/blob/5acaa21b6511c0e43e5f9e6c45da50437a6dca51/specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2020-03-15-preview/storagepool.json#L1338), will that do?